src/components/UI/Banner/Banner.module.css

```diff
@@ -0,0 +1,40 @@
+.banner {
+  display: flex;
+  flex-flow: column;
+  position: relative;
+  height: 95vh;
+  justify-content: center;
+  padding-left: 2rem;
+  gap: 2rem;
+}
+
+.banner-img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 0;
```

Comment:
Suggestion: use variables for z-index for ease of maintaining stacking context at one place.